deploy-hs

```diff
@@ -86,5 +86,9 @@ docker exec $CID python manage.py makemigrations
 echo "*** manage.py migrate ***"
 docker exec $CID python manage.py migrate
 
+docker exec $CID python manage.py migrate hs_core zero
+
+docker exec $CID python manage.py migrate
+
```

Comment:
Your own branch isn't updated when you pull develop, only when you
explicitly merge develop into your own branch.

On Wed, May 13, 2015 at 12:57 PM, Tony Castronova <notifications@github.com>
wrote:

> In deploy-hs
> <https://github.com/hydroshare/hydroshare/pull/346#discussion_r30251460>:
>
> > @@ -86,5 +86,9 @@ docker exec $CID python manage.py makemigrations
> >  echo "*** manage.py migrate ***"
> >  docker exec $CID python manage.py migrate
> >
> > +docker exec $CID python manage.py migrate hs_core zero
> > +
> > +docker exec $CID python manage.py migrate
> > +
>
> This is not a file that I edited. Shouldn't this be updated when I pull
> develop? Am I doing something wrong?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hydroshare/hydroshare/pull/346/files#r30251460>.
>



-- 


*Calvin Spealman*Lead Front-end Developer

calvin@caktusgroup.com <calvin@caktusgroup.com>
www.caktusgroup.com